draft-ietf-masque-h3-datagram.md

```diff
@@ -193,8 +193,8 @@ allow communicating prioritization preferences.
 
 ### The SETTINGS_H3_DATAGRAM HTTP/3 Setting {#setting}
 
-Implementations of HTTP/3 that are willing to receive HTTP Datagrams can
-indicate that to their peer by sending the SETTINGS_H3_DATAGRAM setting with a
+Implementations that are willing to receive HTTP/3 Datagrams can indicate
+support to their peer by sending the SETTINGS_H3_DATAGRAM setting with a
```

Comment:
We specifically removed the term support since that refers to the implementation not the intent
```suggestion
Endpoints can indicate to their peer that they are willing to receive HTTP/3
Datagrams  by sending the SETTINGS_H3_DATAGRAM setting with a
```